ample/modelling/__main__.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ccp4-python
+
+__author__ = "Jens Thomas, and Felix Simkovic"
+__date__ = "22 Mar 2019"
+__version__ = "1.0"
+
+import argparse
+import os
+import logging
+import multiprocessing
+
+from ample.util import argparse_util
+from ample.util import logging_util
+from ample.modelling.rosetta_model import RosettaModel
+
+def set_defaults(args):
```

Comment:
why not set all of these defaults directly to the namespace? either via `default` or by providing an `argparse.Namespace`

Reply:
I'm not sure what you mean. I think the defaults here should be kept separate from the AMPLE defaults, so I don't think using default on the command-line options would work (if that's what you meant?). What do you mean by "providing an argparse.Namespace"?

Reply:
https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.set_defaults

you simply need to provide keyword arguments, it will do the checking automatically.
```python
>>> import argparse
>>> p = argparse.ArgumentParser()
>>> p.add_argument('--name')
>>> p.set_defaults(name='Jens')
>>> args = p.parse_args()
```

Reply:
Oh yes - much neater!